### DISCUSSION TITLE
Cost tracker - track a washing machine run costs past midnight

### DISCUSSION THREAD

**Discussion by McBeats** [BottlecapDave/HomeAssistant-OctopusEnergy#1038] [Q&A]
Hi
I am probably doing this wrong but....

So I have a washing machine with a energy tracking plug on it. I have a tracker sensor linked and it seems to work. When I detect the washing machine running it turns on is_trackable (as the plug uses a slight amount all the time) and when its over it stops that.
Currently though if my wash goes past midnight on Sunday it resets the trackable costs I think (looking now)

Is there a way to get the run cost for the washing machine/dishwasher etc that runs over midnight?

Like I said I am probably doing something wrong but I thought I would ask.

Thanks

**Comment by BottlecapDave** [comment]
Hello. I'm afraid the cost tracker is currently setup to reset at midnight as they've been built as daily, weekly and monthly sensors. You can configure the day the weekly and monthly sensors update when setting up the sensor (or updating the config). If you're needing to configure the time the sensors reset then please raise an enhancement to support this, but no promises on when this will be done (unless you wish to contribute).

**Comment by McBeats** [comment]
Sorry I was being a moron as it was late, I had a umeter tracking it and it was set up a while ago and I forgot. It tracks the price over the time change.
